src/main/kotlin/blackjack/domain/card/CardNumber.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.domain.card
+
+enum class CardNumber(val value: Int) {
+    ONE(1),
```

Comment:
```suggestion
    ACE(1),
```
카드 중에는 1이라는 것은 존재하지 않습니다.
기본적으로 Ace를 1로 사용하고 있으며, 조건에 따라 11이 됩니다. 🙏

Reply:
아하.. 그렇군요 ㅋㅋ... 룰이 생각보다 복잡해서 놓쳤던 것 같네요 감사합니다 ✔️